core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-uve-toolbar/components/dot-ema-bookmarks/dot-ema-bookmarks.component.html

```diff
@@ -1,12 +1,16 @@
 @if (store.$previewMode()) {
     <p-button
+        [pTooltip]="'editpage.toolbar.bookmark' | dm"
```

Comment:
@rjvelazco I realized that the difference between a button and another is the `label` and a `class`. I suggest using one button with these variations in order to `previewMode`, like this:

```
@let previewMode = store.$previewMode();

<p-button
    [pTooltip]="'editpage.toolbar.bookmark' | dm"
    tooltipPosition="bottom"
    (click)="toggleBookmark()"
    [icon]="bookmarked() ? 'pi pi-star-fill' : 'pi pi-star'"
    [loading]="loading()"
    [label]="previewMode ? null : 'editpage.toolbar.bookmark.preview' | dm"
    styleClass="p-button-text p-button-sm p-button-rounded"
    [class.p-button-rounded]="previewMode"
    data-testId="bookmark-button" />

```

Reply:
Done!